src/App.js

```diff
@@ -13,6 +13,7 @@ import ChangePw from 'components/Auth/ChangePw';
 import MainPage from 'pages/mainPage';
 import { setTokenOnHeader } from 'api/logined';
 import { getCookie } from 'components/Shared/Cookies';
+import MyPage from 'pages/myPage';
```

Comment:
이제 봤는데 페이지 폴더 내의 컴포넌트들이 다 camelCase 로 되어있네요.
컴포넌트들은 PascalCase 로 만들겠다는 convention 이 있었는데 파일 이름도 이에 맞게 수정해주는게 어떨까요?

```suggestion
import MyPage from 'pages/MyPage';
```

Reply:
Page 폴더를 컴포넌트라고 안보고 단순 파일이라고 생각해서 ;; 작성한 걸로 보아 PascalCase가 더 적절해 보입니다

Reply:
b66b65e